tests/unit/api/helper.py

```diff
@@ -28,7 +28,7 @@
 
 API_KEY = "apikey"
 APP_KEY = "applicationkey"
-API_HOST = "host"
+API_HOST = "https://api.datadoghq.com"
```

Comment:
Nit: Since these are all unit tests and shouldn't actually interact with the API, can we just make this `"https://example.com"` or something similarly obvious that it isn't a real integration test?

Reply:
Updated